### PR TITLE
Concourse deploy

### PR DESCRIPTION
**Pull request by jmcarp**
- Deploy to staging
- Run acceptance tests on staging
- Deploy to production
- Run acceptance tests on production

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -3,5 +3,5 @@
 set -e -x
 
 cd s3-broker-app
-mvn package
+mvn package -Dmaven.test.skip=true
```
Comment (afeld) [inline ci/build.sh]: I thought this was temporary...? We decided we did want to run the unit/integration as a prerequisite to deploy, right?

**Comment by afeld** [comment]
Just the one question—looks fine otherwise!

**Comment by afeld** [comment]
Merge when ready!

**Comment by afeld** [comment]
Closes https://github.com/18F/cg-atlas/issues/44.

**Comment by ghost** [comment]
LGTM

**Pull request merged**